prompt.go

```diff
@@ -1,16 +1,20 @@
 package prompt
 
-import "github.com/howeyc/gopass"
-import "strings"
-import "strconv"
-import "fmt"
+import (
```

Comment:
I seem that this PR made goxc failed with ubuntu, I will try to use bufio scanner